sqlglot/dialects/snowflake.py

```diff
@@ -291,7 +291,8 @@ class Parser(parser.Parser):
         FUNCTIONS = {
             **parser.Parser.FUNCTIONS,
             "APPROX_PERCENTILE": exp.ApproxQuantile.from_arg_list,
-            "ARRAYAGG": exp.ArrayAgg.from_arg_list,
+            "ARRAYAGG": lambda args: exp.ArrayAgg(this=seq_get(args, 0), nulls_excluded=True),
+            "ARRAY_AGG": lambda args: exp.ArrayAgg(this=seq_get(args, 0), nulls_excluded=True),
```

Comment:
Any use in moving the flag to the dialect and setting a builder for `ARRAY_AGG` & it's aliases, which will set `nulls_excluded=self.ARRAY_AGG_INCLUDES_NULL`?

Reply:
[Done](https://github.com/tobymao/sqlglot/pull/4033/commits/59265d1d2fbdbb1ab045af9520d8ae60c1f44c3a)